src/common/stores/advertisementStore.ts

```diff
@@ -0,0 +1,14 @@
+import { ref, computed } from 'vue'
+import { defineStore } from 'pinia'
+import { User } from '../models/advertisementModels'
+
+export const useUserStore = defineStore('user', () => {
```

Comment:
Why is this even needed? Is it even used anywhere?

Reply:
nope, removed